src/glados/TTS/tts_glados.py

```
@@ -231,35 +196,13 @@ def generate_speech_audio(self, text: str) -> NDArray[np.float32]:
-        Convert a list of phoneme sentences to synthesized audio.
-
-        Generates audio for each phoneme sentence and concatenates the results into a single audio array.
-
-        Parameters:
-            phonemes (list[str]): A list of phoneme sentences to convert to speech.
-
-        Returns:
-            NDArray[np.float32]: A numpy array containing the synthesized audio, with each sentence concatenated.
-            Returns an empty float32 numpy array if no audio could be generated.
+        phoneme_ids_list = [self._phonemes_to_ids(sentence) for sentence in phonemes]
+        audio_chunks = [self._synthesize_ids_to_audio(phoneme_ids) for phoneme_ids in phoneme_ids_list]
```

Pylint (3.3.7)

[convention] Line too long (103/100)

(C0301)